playbooks/common/openshift-cluster/upgrades/pre/verify_upgrade_targets.yml

```diff
@@ -31,6 +31,13 @@
     register: avail_openshift_version
     when: not openshift.common.is_containerized | bool
 
+  - name: Fail when unable to determine available OpenShift RPM version
```

Comment:
I had made a bunch of changes to this file to standardize stuff, including putting in blocks, but then backed it all out because I didn't want to conflate the actual change I was making to put in the one `fail` task.  I'll go ahead and make the other standardization changes.